pkg/monitor/intervalcreation/types.go

```diff
@@ -2,19 +2,24 @@ package intervalcreation
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"path/filepath"
 	"sort"
 	"time"
 
 	"github.com/openshift/origin/pkg/monitor/apiserveravailability"
-
 	"github.com/openshift/origin/pkg/monitor/nodedetails"
+	monitorserialization "github.com/openshift/origin/pkg/monitor/serialization"
 
 	"github.com/openshift/origin/pkg/monitor/monitorapi"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
+const InClusterEventsFile = "junit/in-cluster-events.json"
```

Comment:
If you could do me a solid and rename any use of "events" to "intervals" in this PR, I would appreciate, I'm trying to drift the code in this direction even though it's still inconsistently used, events is too overloaded.

Reply:
Renamed into "intervals"